app/src/lib/design-system/components/RQNetworkTable/GenericNetworkTable/index.tsx

```diff
@@ -71,17 +73,34 @@ export const GenericNetworkTable = <NetworkLog,>({
     [networkEntrySelector]
   );
 
+  const filterLog = (networkLog: NetworkLog) => {
+    const entry = networkEntrySelector(networkLog);
+
+    if (filters.search) {
+      if (!entry?.request?.url?.includes(filters.search)) {
+        return false;
+      }
+    }
+
+    return true;
+  };
```

Comment:
Note: `includes` will match the empty string

```suggestion
  const filterLog = useCallback((networkLog: NetworkLog) => {
    const entry = networkEntrySelector(networkLog);
    return !!entry?.request?.url?.includes(filters.search))
  },[])
```